### DISCUSSION TITLE
Update Week Start Day options when Language is changed

### DISCUSSION THREAD

**Discussion by missmatsuko** [craftcms/cms#8985] [Ideas]
### Description

I think the week start date values should be updated when the formatting locale is changed. 

Currently it only updates after saving and refreshing the page:
<img width="433" alt="Screenshot of Formatting Date and Week Start Day fields." src="https://user-images.githubusercontent.com/17884198/102817003-cb913500-4383-11eb-8891-b6a242bd3f61.png">

**Comment by brandonkelly** [comment]
Now that week day names are going to stick with the language, not the formatting locale (https://github.com/craftcms/cms/issues/7312#issuecomment-754322222), changing the formatting locale shouldn’t have any effect on the Week Start Day field. However this is still valid in the event the user changes their Language preference.
